### PR TITLE
Support labels for GitLab merge requests

### PR DESCRIPTION
**Pull request by zubivan**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Brings support for label to GitLab merge requests.

### :arrow_heading_down: What is the current behavior?

We don't support it.

### :new: What is the new behavior (if this is a feature change)?

We support it.

### :boom: Does this PR introduce a breaking change?

Nope.

### :bug: Recommendations for testing

Try to create a gitlab MR.

### :memo: Links to relevant issues/docs

Already supported by the API. https://docs.gitlab.com/ee/api/merge_requests.html#create-mr
Just never bother to implement it.

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Relevant documentation was updated

### CODE REVIEW

File: NuKeeper.Gitlab/Model/MergeRequest.cs [hunk 1]
```diff
@@ -21,5 +21,8 @@ public class MergeRequest
 
         [JsonProperty("remove_source_branch")]
         public bool RemoveSourceBranch { get; set; }
+
+        [JsonProperty("lables")]
```
Comment (MaxMommersteeg) [inline NuKeeper.Gitlab/Model/MergeRequest.cs]: I think this should be `labels`.

**Comment by MaxMommersteeg** [comment]
Small typo, but LGTM once fixed.

File: NuKeeper.Gitlab/Model/MergeRequest.cs (hunk 1 above)
Comment (zubivan) [reply NuKeeper.Gitlab/Model/MergeRequest.cs]: Haha, damn, I was quite sure to copy it. Silly, thanks for taking a look, Max!

Comment (MaxMommersteeg) [reply NuKeeper.Gitlab/Model/MergeRequest.cs]: No problem, that is why we do code reviews :)

Comment (zubivan) [reply NuKeeper.Gitlab/Model/MergeRequest.cs]: And that's fixed... 🤦‍♂

**Comment by MarcBruins** [comment]
Thanks Ivan!

**Pull request merged**